xarray/core/dataarray.py

```diff
@@ -1220,6 +1220,93 @@ def fillna(self, value):
         out = ops.fillna(self, value)
         return out
 
+    def interpolate_na(self, dim=None, method='linear', limit=None,
+                       use_coordinate=True,
+                       **kwargs):
+        """Interpolate values according to different methods.
+
+        Parameters
+        ----------
+        dim : str
+            Specifies the dimension along which to interpolate.
+        method : {'linear', 'nearest', 'zero', 'slinear', 'quadratic', 'cubic',
+                  'polynomial', 'barycentric', 'krog', 'pchip',
+                  'spline'}, optional
+            String indicating which method to use for interpolation:
+
+            - 'linear': linear interpolation (Default). Additional keyword
```

Comment:
I noticed `pandas.interpolate(method='linear')` behaves differently.
Though I think our definition is more natural,  we can note this here?

Reply:
Are you referring to which values Pandas uses for the index?

Reply:
Yes.

pandas' [docstring](https://pandas.pydata.org/pandas-docs/stable/generated/pandas.Series.interpolate.html) says 
> ‘linear’: ignore the index and treat the values as equally spaced.
> ‘index’: use the actual numerical values of the index.   

the difference of which may correspond to our `use_coordinate` option [False/True].

Our `linear` option means a linear interpolation. 
I've just thought the same name with different meanings is sometimes confusing.
It is a very trivial point (and maybe a kind of overconcern).